libbindgen/src/ir/context.rs

```diff
@@ -431,6 +431,18 @@ impl<'ctx> BindgenContext<'ctx> {
         // because we remove it before the end of this function.
         self.gen_ctx = Some(unsafe { mem::transmute(&ctx) });
 
+        // NOTE (imp): I guess this should work but how do I test it?
+        for (id, item) in self.items() {
+            let mut sub_items = ItemSet::new();
```

Comment:
On top of that, can we call this `related_items` instead? Seems more accurate given we insert the parent.